fhir-server-test/src/test/java/com/ibm/fhir/server/test/FHIRNotificationServiceClientEndpoint.java

```diff
@@ -53,8 +53,8 @@ public void onOpen(Session session, EndpointConfig config) {
         session.addMessageHandler(new MessageHandler.Whole<String>() {
             public void onMessage(String text) {
                 System.out.println(">>> Received message: " + text);
-                FHIRNotificationEvent event = FHIRNotificationUtil.toNotificationEvent(text);
-                events.add(event);
+            //    FHIRNotificationEvent event = FHIRNotificationUtil.toNotificationEvent(text);
```

Comment:
why is this line commented out?  is it safe to remove?

Reply:
safe to remove, I was wondering if I should add comments about reason for the change. let me remove

Reply:
done